### PR TITLE
Added browserlist settings to development documentation

### LINKED ISSUE #407: Developer environment documentation is missing browserlist for autoprefixer
# Bug report

- [x] I’ve read and understood the [Contributing guidelines](https://github.com/govau/design-system-site/blob/master/CONTRIBUTING.md) and have done my best effort to follow them.
- [x] I’ve read and agree to the [Code of Conduct](https://github.com/govau/design-system-site/blob/master/CODE_OF_CONDUCT.md).
- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

When following the documentation you may experience some odd issues with css rendering related to autoprefixer not being ran on certain browsers. This is because we are missing our `browserlist`:

https://github.com/govau/design-system-site/blob/738e4c3b4a08dd1e7e82bc6fca809799e553a2a7/package.json#L149-L154

## What I expected to happen

Documentation should include `browserlist`

## Reproducing

- Website URL: https://designsystem.gov.au/get-started/development-environment/

### PR DESCRIPTION
**Pull request by adamzerella**
Adds a note to append `browserlist` to the development `package.json`.

### CODE REVIEW

File: content/get-started/development-environment/content.md [hunk 1]
```diff
@@ -76,6 +76,16 @@ To do this we edit the `scripts` in the `package.json` file:
   "watch:sass": "onchange './src/sass/**/*.scss' -- npm run build",
 }
 ```
+and make sure we let our modules know what browsers we are supporting:
```
Comment (sukhrajghuman) [inline content/get-started/development-environment/content.md]:
```suggestion
Autoprefixer uses [Browserslist](https://www.npmjs.com/package/browserslist), so you can specify the browsers you want to target with the `browserlist` key in the `package.json` file like so:
```

Comment (adamzerella) [reply content/get-started/development-environment/content.md]: I'm happy to change it but I want to somehow say that `browserslist` is used for a lot of other modules

Comment (sukhrajghuman) [reply content/get-started/development-environment/content.md]: why?

**Comment by adamzerella** [comment]
After some discussion with @sukhrajghuman we've agreed that in this context `browserlist` is only relevant to autoprefixer.

**Pull request merged**